### ISSUE TITLE
Remove exported names

### ISSUE THREAD

**Issue by fredrikekre** [Ferrite-FEM/Ferrite.jl#746]
@lijas, @kimauth, and I looked over the export list and think we should remove and/or change the following:
```
# Remove from export
getweights
getpoints

# Can be documented but don't have to be exported (used in e.g. FerrieViz to export all sets?)
getcellsets
getnodesets
getfacesets
getedgesets
getvertexsets

# Risky to use unless we compute the matrix on construction
onboundary

# Very generic name, rename to e.g. transform_coordinates!?
transform!

# Rename to finish_assemble to match start_assemble?
end_assemble

# Maybe rename to evaluate_at_points (we have evaluate_at_grid_nodes which does something similar)
get_point_values
```

**Comment by termi-official** [comment]
Do you really want to hardly remove them or deprecate? I especially think that `onboundary` might be used in user code, because we have it in the examples (although I agree that it is not super great).

**Comment by fredrikekre** [comment]
Closes by #754.
